Source/Turbo/Navigator/Helpers/Navigation.swift

```diff
@@ -18,5 +18,7 @@ public enum Navigation {
         case medium
         case large
         case full
+        case pageSheet
+        case formSheet
```

Comment:
I hate to come in late after a merge, but I think these should be:

```swift
case pageSheet = "page_sheet"
case formSheet = "form_sheet"
```

to match `Navigation.Presentation.clearAll` and `Navigation.Presentation.replaceRoot`.